client/X11/xf_gdi.c

```diff
@@ -1030,24 +1030,24 @@ static BOOL xf_gdi_surface_bits(rdpContext* context,
 			break;
 
 		case RDP_CODEC_ID_NSCODEC:
-			format = FREERDP_VFLIP_PIXEL_FORMAT(gdi->dstFormat);
+			format = gdi->dstFormat;
```

Comment:
Isn't the format variable quite useless now?
I think we can just pass gdi->dstFormat (or PIXEL_FORMAT_BGRX32 in the next case) to the functions instead.